AnkiDroid/src/main/java/com/ichi2/anki/dialogs/TtsPlaybackErrorDialog.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (c) 2023 David Allison <davidallisongithub@gmail.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify it under
+ *  the terms of the GNU General Public License as published by the Free Software
+ *  Foundation; either version 3 of the License, or (at your option) any later
+ *  version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ *  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ *  PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with
+ *  this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.ichi2.anki.dialogs
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.provider.Settings
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.fragment.app.DialogFragment
+import com.ichi2.anki.R
+import com.ichi2.anki.showThemedToast
+import com.ichi2.anki.utils.openUrl
+
+class TtsPlaybackErrorDialog(private val language: String, private val engine: String) : DialogFragment() {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.dialog_tts_error, container, false)
+
+        val reasonTextView = view.findViewById<TextView>(R.id.reason_text)
+        val reasonText = getString(R.string.tts_error_dialog_reason_text, engine, language)
+        reasonTextView.text = reasonText
+
+        val showSupportedVoicesButton = view.findViewById<Button>(R.id.second_button)
+        showSupportedVoicesButton.setOnClickListener {
+            showVoicesDialog()
+        }
+
+        val openSettingButton = view.findViewById<Button>(R.id.first_button)
+        openSettingButton.setOnClickListener {
+            openSettings()
+        }
```

Comment:
A number of lines can be simplified

```suggestion
       view.findViewById<Button>(R.id.first_button).setOnClickListener {
            openSettings()
        }
```